modules/test/playwright/pages/product-navigation-user-personal-bar-web/UserPersonalBarPage.ts

```diff
@@ -79,24 +108,48 @@ export class UserPersonalBarPage {
 		await this.applicationsMenuPage.goToInstanceSettings();
 		await this.usersSetting.click();
 		await this.showNotificationBadgeInPersonalMenuLabel.check();
-		await this.editConfigurationSubmitButton.click();
+		await Promise.all([
+			this.editConfigurationSubmitButton.click(),
+			this.page.waitForResponse(
+				(resp) =>
+					resp.status() === 200 &&
+					resp
+						.url()
+						.includes(
+							'p_p_id=com_liferay_configuration_admin_web_portlet_InstanceSettingsPortlet'
+						)
+			),
+		]);
 	}
 
 	async enableSingleApproverWorkflowProduct() {
 		await this.goToProcessBuilderConfigurationTab();
+		await this.searchInput.fill('Product');
+		await this.searchSubmit.click();
+		await this.page.reload();
+		await this.workflowDefinitionLinkEditButton.waitFor({
+			state: 'attached',
+		});
```

Comment:
I need to talk to you about this implementation tomorrow because the button does not consistently work when the button is clicked.